docs/networking.md

```diff
@@ -54,6 +54,7 @@ One goal of k0s is to allow for the deployment of an isolated control plane, whi
 | TCP      | 10250 | kubelet        | controller, worker => host `*` | Authenticated kubelet API for the controller node `kube-apiserver` (and `heapster`/`metrics-server` addons) using TLS client certs
 | TCP      | 9443  | k0s-api        | controller <-> controller      | k0s controller join API, TLS with token auth
 | TCP      | 8132  | konnectivity   | worker <-> controller          | Konnectivity is used as "reverse" tunnel between kube-apiserver and worker kubelets
+| TCP      | 112   | keepalived     | controller <-> controller      | Only required for control plane load balancing vrrpInstances for ip address 224.0.0.18.
```

Comment:
I don't get the reference to a specific IP address?

Reply:
It's a multicast IP address. VRRP defines it in [RFC 3768](https://datatracker.ietf.org/doc/html/rfc3768#section-5.2.2):

[5.2.2](https://datatracker.ietf.org/doc/html/rfc3768#section-5.2.2).  Destination Address

   The IP multicast address as assigned by the IANA for VRRP is:

      224.0.0.18

   This is a link local scope multicast address.  Routers MUST NOT
   forward a datagram with this destination address regardless of its
   TTL.

I will add a note in the documentation.